sass/tools/_modes.scss

```diff
@@ -0,0 +1,47 @@
+/// # If-Mode
+/// @group tools-modes
+
+// If-Mode
+// -------
+/// Toggles between two number values
+/// (or CSS `var()` functions that return numbers),
+/// depending on the current light/dark mode.
+///
+/// @param {number | var()} $light -
+///   The value to use for light mode
+/// @param {number | var()} $dark -
+///   The value to use for dark mode
+/// @return {calc()} -
+///   A CSS `calc()` function that will toggle the proper values
+///   based on the value of `--ccs-mode--zero` and `--ccs-invert--zero`
+/// @example scss -
+///   [data-ccs-theme] {
+///     --ccs-accent--theme: #{tools.if-mode(
+///       $light: var(--ccs-theme--1),
+///       $dark: var(--ccs-theme--2)
+///     )};
+///     --ccs-special--theme: #{tools.if-mode(
+///       $light: var(--ccs-theme--2),
+///       $dark: var(--ccs-theme--1)
+///     )};
+///   }
+/// @example scss -
+///   [data-ccs-theme='oddbird'] {
+///     --ccs-prime--config: 195;
+///     --ccs-accent--config: 330;
+///     --ccs-prime--theme: #{tools.if-mode(
+///       $light: var(--ccs-prime--config),
+///       $dark: var(--ccs-accent--config)
+///     )};
+///     --ccs-accent--theme: #{tools.if-mode(
+///       $light: var(--ccs-accent--config),
+///       $dark: var(--ccs-prime--config)
+///     )};
+///     --ccs-custom-hue: none;
+///   }
+/// @group tools-modes
+@function if-mode($light, $dark) {
+  @return calc(
+    var(--ccs-mode--zero) * #{$light} + var(--ccs-invert--zero) * #{$dark}
+  );
+}
```

Comment:
@mirisuzanne Is this file a duplicate of `partials/_tools.scss`?

Reply:
oh, it's because you made changes to one, after I had moved it. Fixing…